ui/Decoder.elm

```diff
@@ -13,7 +13,7 @@ buildRecordDecoder =
         |> required "build" buildDecoder
         |> required "download" downloadDecoder
         |> required "source" sourceDecoder
-        |> required "systemaddons" (nullable (list systemAddonDecoder))
+        |> required "systemaddons" (maybe (list systemAddonDecoder))
```

Comment:
I believe the other similar changes were for similar issues? Maybe we also need to change the decoders in those cases?